Emby.Naming/AudioBook/AudioBookNameParserResult.cs

```diff
@@ -14,5 +16,50 @@ public struct AudioBookNameParserResult
         /// Gets or sets optional year of release.
         /// </summary>
         public int? Year { get; set; }
+
+        /// <summary>
```

Comment:
i am generally not a huge fan of overloading operators if you dont have a very good reason for it

Reply:
I'm confused. This is from your commit: https://github.com/jellyfin/jellyfin/pull/12401/files#diff-ae9b100c09870f12039b99d5fa71e295a6ba55ac78e363b54270bdd58f8c9f51. I have no opinion on it. I was solely trying to help to move your PR forward. If you no longer want to move forward with those changes, we can certainly close both. I'll look for another stale PR to try to land.

Here are the changes I made (to fix build breaks, after resolving the merge conflicts): https://github.com/jellyfin/jellyfin/pull/13693/commits/099e994cae1d8127fbe30a0889dc8223b01f8335

Reply:
you are not wrong.
But that Pr was quite a long time ago and was in all honesty a bit low effort to begin with.